src/containers/TriggerBinding/TriggerBinding.js

```diff
@@ -0,0 +1,249 @@
+    }
+
+    const headersForParameters = [
+      {
+        key: 'name',
+        header: intl.formatMessage({
+          id: 'dashboard.tableHeader.name',
+          defaultMessage: 'Name'
+        })
+      },
+      {
+        key: 'value',
+        header: intl.formatMessage({
+          id: 'dashboard.tableHeader.value',
+          defaultMessage: 'Value'
+        })
+      }
+    ];
+
+    const rowsForParameters = triggerBinding.spec.params.map(
+      ({ name, value }) => ({
+        id: name,
+        name,
+        value
+      })
+    );
+
+    const emptyTextMessage = intl.formatMessage({
+      id: 'dashboard.triggerBinding.noParams',
+      defaultMessage: 'No parameters found for this TriggerBinding.'
+    });
+
+    return (
+      <div className="triggerBinding">
+        <h1>{triggerBindingName}</h1>
+        <Tabs selected={0}>
+          <Tab
+            label={intl.formatMessage({
+              id: 'dashboard.resource.detailsTab',
+              defaultMessage: 'Details'
+            })}
+          >
+            <div className="details">
+              <p>
+                <span>
+                  {intl.formatMessage({
+                    id: 'dashboard.metadata.dateCreated',
+                    defaultMessage: 'Date Created:'
+                  })}
+                </span>
+                <FormattedDate
+                  date={triggerBinding.metadata.creationTimestamp}
+                  relative
+                />
+              </p>
+              <p>
+                <span>
+                  {intl.formatMessage({
+                    id: 'dashboard.metadata.labels',
+                    defaultMessage: 'Labels:'
+                  })}
+                </span>
+                {formattedLabelsToRender.length === 0
+                  ? intl.formatMessage({
+                      id: 'dashboard.metadata.none',
+                      defaultMessage: 'None'
+                    })
+                  : formattedLabelsToRender.map(label => (
+                      <Tag type="blue" key={label}>
+                        {label}
+                      </Tag>
+                    ))}
+              </p>
+              <p>
+                <span>
+                  {intl.formatMessage({
+                    id: 'dashboard.metadata.namespace',
+                    defaultMessage: 'Namespace:'
+                  })}
+                </span>
+                {triggerBinding.metadata.namespace}
+              </p>
+              <Table
+                title={intl.formatMessage({
+                  id: 'dashboard.triggerBinding.parametersTableTitle',
+                  defaultMessage: 'Parameters'
+                })}
+                headers={headersForParameters}
+                rows={rowsForParameters}
+                selectedNamespace={selectedNamespace}
+                emptyTextAllNamespaces={emptyTextMessage}
+                emptyTextSelectedNamespace={emptyTextMessage}
+                isSortable
```

Comment:
let's leave out the sorting for consistency with other pages until we can address https://github.com/tektoncd/dashboard/issues/842